.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -42,11 +42,12 @@ jobs:
 
   run-updater:
     needs:
-      - build-updater
+    - build-updater
     runs-on: ubuntu-latest
     env:
       STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
       STACKROX_NVD_API_CALL_INTERVAL: 6s
+      MANUAL_URL: "https://raw.githubusercontent.com/stackrox/stackrox/d1b557c2dbb21501fd839002fcec083a2d63e8c8/scanner/updater/manual/vulns.yaml"
```

Comment:
Hey, @daynewlee I just realized, do you think this would work if we use `${{ github.event.pull_request.head.sha }}` here? When running on `schedule` this would be master's HEAD, as far as I know, and if dev it would be the top commit of the PR, which is what we want. It would allow us to enable "dev vuln builds" with any changes made in the manual vulnerabilities in the same PR, right?